web/index.html

```diff
@@ -21,15 +18,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <div id="i be trippin"></div>
+    <script type="module" src="/src/index.tsx"></script>
```

Comment:
Is this a change required by vite?

Reply:
yep. it's ok?

Reply:
Yeah just curious if you have a link to the docs about it or some background info.

Reply:
I think so, just wondering if there's any background info on why

Reply:
the beginning of this is good intro. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules Vite is all about using modern native  esm support.